recipes/cc65/all/test_package/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+import shutil
+
+
+class TestPackageConan(ConanFile):
+
+    exports_sources = "hello.c", "text.s"
+
+    _targets = ("c64", "apple2")
+
+    def build(self):
+        for src in self.exports_sources:
+            shutil.copy(os.path.join(self.source_folder, src), os.path.join(self.build_folder, src))
+        try:
+            os.unlink("c64.lib")
```

Comment:
maybe some commend why is it required to remove the file
(or remove that code part, if irrelevant)

Reply:
This is dead code. It can be removed.
Thanks for the check.